resources/standards-data/ldml-keyboards/fixup-schema.js

```diff
@@ -85,6 +85,8 @@ if (data.title.endsWith('ldmlKeyboard3.xsd')) {
       delete item.items.type;
       item.items['$ref'] = `#/definitions/${newName}`;
     }
+    // Workaround CLDR-18138 by not making 'cldr' the default for the base
+    delete data.definitions?.import?.properties?.base?.enum;
```

Comment:
Is this needed?

Reply:
it's only needed until the upstream fix goes through.